rpi_backlight/utils.py

```diff
@@ -1,9 +1,33 @@
+from __future__ import annotations
 from pathlib import Path
 from tempfile import TemporaryDirectory
+from typing import Optional, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from __init__ import BoardType
 
 __all__ = ["FakeBacklightSysfs"]
 
 
+def detect_board_type(boardtype: "BoardType") -> Optional[boardtype]:
```

Comment:
Not sure why this has a parameter now? What I meant is something like this:

```python3
if TYPE_CHECKING:
    from . import BoardType
```

and then

```python3
def detect_board_type() -> Optional[BoardType]:
    from . import BoardType

    model_file = Path("/proc/device-tree/model")
    ...
```

Reply:
Thanks, in my Googling the example was like that.
https://stackoverflow.com/questions/65619107/getting-nameerror-for-annotation-even-if-i-use-type-checking-import
I will reupload with your changes

Thanks,